arrows/proj/tests/test_geo_map.cxx

```diff
@@ -1,149 +0,0 @@
-/*ckwg +29
- * Copyright 2013-2016 by Kitware, Inc.
- * All rights reserved.
- *
- * Redistribution and use in source and binary forms, with or without
- * modification, are permitted provided that the following conditions are met:
- *
- *  * Redistributions of source code must retain the above copyright notice,
- *    this list of conditions and the following disclaimer.
- *
- *  * Redistributions in binary form must reproduce the above copyright notice,
- *    this list of conditions and the following disclaimer in the documentation
- *    and/or other materials provided with the distribution.
- *
- *  * Neither name of Kitware, Inc. nor the names of any contributors may be used
- *    to endorse or promote products derived from this software without specific
- *    prior written permission.
- *
- * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS ``AS IS''
- * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
- * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
- * ARE DISCLAIMED. IN NO EVENT SHALL THE AUTHORS OR CONTRIBUTORS BE LIABLE FOR
- * ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
- * DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
- * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
- * CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
- * OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
- * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
- */
-
-/**
- * \file
- * \brief Test PROJ geo map functionality
- */
-
-#include <test_common.h>
-
-#include <vital/plugin_loader/plugin_manager.h>
-#include <arrows/proj/geo_map.h>
-
-// test_common.h required things
-#define TEST_ARGS ()
-DECLARE_TEST_MAP();
-
-// distance epsilons for centimeter accuracy or better
-static const double met_epsilon = 0.01;
-static const double deg_epsilon = 0.0000001;
-
-int main(int argc, char** argv)
-{
-  CHECK_ARGS(1);
-
-  testname_t const testname = argv[1];
-
-
-  RUN_TEST(testname);
-}
-
-
-IMPLEMENT_TEST(factory)
-{
-  kwiver::vital::plugin_manager::instance().load_all_plugins();
-
-  kwiver::vital::algo::geo_map_sptr gmap = kwiver::vital::algo::geo_map::create("proj");
-  if (!gmap)
-  {
-    TEST_ERROR("Unable to create geo_map algorithm of type proj");
-  }
-  kwiver::vital::algo::geo_map* gmap_ptr = gmap.get();
-  if (typeid(*gmap_ptr) != typeid(kwiver::arrows::proj::geo_map))
-  {
-    TEST_ERROR("Factory method did not construct the correct type");
-  }
-}
-
-
-IMPLEMENT_TEST(latlon_to_utm)
```

Comment:
Does it make sense to adapt some of these tests to use the new API rather than remove the tests entirely?  It would be good to have some unit tests for geo conversions somewhere in KWIVER.